Sources/Testing/Testing.docc/known-issues.md

```diff
@@ -0,0 +1,38 @@
+# Known issues
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2023 Apple Inc. and the Swift project authors
```

Comment:
```suggestion
Copyright © 2023–2024 Apple Inc. and the Swift project authors
```

Reply:
With copyright statements, it's important to note the years that you create or edit a page. So here, 2023–2024 is applicable. If you don't touch this until 2026, you'd use something like, "2023–2024, 2026" to denote the years. (Just FYI.)